<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>543</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>834</width>
    <height>543</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Choose Task</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>Task1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Task2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Task3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Task4</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxTask3">
      <property name="title">
       <string>Task3</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Text From txt File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditTask3txt"/>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Text From bin File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditTask3bin"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxTask1">
      <property name="title">
       <string>Task1</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Text Add To File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditTask1"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Text From File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditTask1"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxTask2">
      <property name="title">
       <string>Task2</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Size of array</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxSize">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Number</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxNumber">
         <property name="minimum">
          <number>-99</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonAddNember">
         <property name="text">
          <string>Add Number</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Text Add To File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditTask2"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Text From File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditTask2"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxTask4">
      <property name="title">
       <string>Task4</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Text Add To File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditTask4"/>
       </item>
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Text From File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditTask4"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxMotorbike">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>1000</height>
       </size>
      </property>
      <property name="title">
       <string>Your Motorbike</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Serial Number</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxSerialNumber">
         <property name="minimum">
          <number>100000</number>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Max Speed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBoxMaxSpeed"/>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Color(Only First Symbol like R = Red)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditColor"/>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Name(less than 10 symbols)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditName"/>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>isMoving</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Length</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxLength"/>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Width</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxWidth"/>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Height</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxHeight"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonAddBike">
         <property name="text">
          <string>Add my Motorbike</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>834</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
